### PR TITLE
added ExtraTreesClassifier_MaxAbsScaler.ipynb

### PR DESCRIPTION
**Pull request by shreepad-nade**

### CODE REVIEW

**Comment by Thilakraj1998** [comment]
The MaxAbsScaler initial has some issues
change it like this:

maxabsscaler= MaxAbsScaler()
X_train=maxabsscaler.fit_transform(X_train)
X_test=maxabsscaler.transform(X_test)

this will ensure consistence between the train and test set
rather then creating 2 separate initialization for train and test
